js/views/roomlistview.js

```diff
@@ -290,22 +290,23 @@
 		leaveGroup: function() {
 			//If user is in that room, it should leave that room first.
 			if (this.model.get('active')) {
-				OCA.SpreedMe.Rooms.leaveCurrentRoom();
-				OCA.SpreedMe.Rooms.showRoomDeletedMessage(true);
+				OCA.SpreedMe.Calls.leaveCurrentCall(true);
+			} else {
+				// TODO(fancycode): When does this happen? The user is in a call
+				// where the call is not active?
+				$.ajax({
+					url: OC.linkToOCS('apps/spreed/api/v1/room', 2) + this.model.get('token') + '/participants/self',
```

Comment:
The ajax call is in `leaveCall` in the signaling code (you just commented on that), which is triggered by SimpleWebRTC when a user leaves a call. Again probably some room vs. call confusion on my side ;-)